translate/storage/pypo.py

```diff
@@ -770,6 +773,10 @@ def parse(self, input):
         self.units = []
         poparser.parse_units(poparser.ParseState(input, pounit), self)
 
+        # duplicates are now removed by default unless duplicatestyle=allow
+        if duplicatestyle != "allow":
+            self.removeduplicates(duplicatestyle=duplicatestyle)
+
     def removeduplicates(self, duplicatestyle="merge"):
```

Comment:
Can you please expand the docstring to list all the acceptable `duplicatestyle` values?